tests/org.eclipse.emfcloud.modelserver.edit.tests/build.properties

```diff
@@ -1,5 +1,5 @@
 source.. = src/
-output.. = bin/
+output.. = target/classes/
```

Comment:
can we leave this on the eclipse standard and keep `bin`?

Reply:
I don't mind either way, as long as .classpath and build.properties are consistent :) all other plug-ins currently use target/classes/

Reply:
me neither, as long as it is consistent